salt/metalk8s/kubernetes/kubelet/standalone.sls

```diff
@@ -8,6 +8,12 @@
 include:
   - .running
 
+Ensure resolv config file exists:
+  file.managed:
+    - name: {{ kubelet.config.resolvConf }}
+    - create: true
+    - replace: false
```

Comment:
Can you explain this part? I see we're defining it to be `/etc/resolv.conf`, and we're passing it to kubelet, but not sure what it does... And also is it safe to just create an empty one? Like, how would this interact with the rest of the system services?

Reply:
It just creates an empty `resolv.conf` because Kubelet needs a file otherwise it fails, and for my testing it works well with an empty file

Reply:
OK, let's keep the conversation open for future readers, but I trust your testing 😉